index.html

```diff
@@ -90,7 +90,234 @@ <h1 class="primary-text-header">I’m Steven Levoyer to see you!</h1>
     </header>
     <!--End-Header-->
     <!--Main-->
+    <main>
+        <!--Works-->
+        <section aria-label="Section of works">
+            <div class="work">
+                <div class="card-works">
+                    <img class="snapshoot" src="./img/Snapshoot-1.svg">
+                    <div class="left-block">
+                        <div class="primary-text-work">
+                            <h2 class="project-title">Tonic</h2>
+                            <div class="frame-2">
+                                <div class="client">
+                                    <p class="text-p">CANOPY</p>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="role">
+                                    <p class="text-p">Back End Dev</p>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="year">
+                                    <p class="text-p">2015</p>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="secondary-text-work">
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        </div>
+                        <div class="tags">
+                            <ul class="list-tag">
+                                <li class="tag"><a href="#" class="text-s">html</a></li>
+                                <li class="tag"><a href="#" class="text-s">css</a></li>
+                                <li class="tag"><a href="#" class="text-s">javaScript</a></li>
+                            </ul>
+                        </div>
+                        <div class="action">
+                            <button type="button" class="action-button">
+                                <p class="text-a">See Project</p>
+                            </button>
```

Comment:
- [ ] Kindly add a pressed and hover state to your buttons to match the design. You can use the `:hover` and `:active` pseudo-classes for this.
![btn1](https://user-images.githubusercontent.com/94551484/213791279-8dab7576-6346-417a-bd86-09e084df6e4c.png)